### ISSUE TITLE
Verbose feedback user warning if PP3 does not apply as expected

### ISSUE THREAD

**Issue by Beep6581** [Beep6581/RawTherapee#4261]
RawTherapee should provide verbose user feedback if some element of the processing profile fails to load, for example:
- A flat-field image was not found - show the filename.
- A dark-frame image was not found - show the filename.
- The PP3 uses a tool which does not exist - show the keys.
- The HaldCLUT could not be found, e.g. #4388 - show the filename.
- The input/working/output profile is not found (e.g. #4478) - show the filename.
- The input/working/output profile is found but is not identical (PP3 could store the hash).
- An input profile could not be loaded, e.g. #4260 - show the filename.
- Raw decoding parameters changed in `camconst.json` (a PP3 is applicable to any camera make/model, so this is a difficult one).

This could be done for example using a Gtk::InfoBar, or a log textbox panel.

P.S. I thought we already had an issue for this but I can't find it...

**Comment by Beep6581** [comment]
I again ran into an issue where I thought my PP3s were broken or I was somehow using the wrong PP3s, until I realized the HaldCLUT was not being found as specified in the sidecar PP3.
We should implement this one way or another for 5.5

**Comment by heckflosse** [comment]
I agree that we should do something here. But I think we shouldn't do it for 5.5. Ok to set milestone to 5.6?

**Comment by Beep6581** [comment]
Yes, it's time for a new release.
